### ISSUE TITLE
Dysfunctional "Reset" button

### ISSUE THREAD

**Issue by Timbobeek** [vkmicro/GB_IssueTracking#3]
**Describe the suggestion / bug**
"Reset" button is dysfunctional, the button changes color when tapped, but nothing happens.

**Steps to reproduce the behavior**
1. Tap on "New Routine"
2. Tap on "Add Exercise"
3. Tap on "Add Exercise"
4. Hit "Reset"

**Actual Result**
Button changes color, but nothing happens after

**Expected Result**
The button triggers some kind of a system reset/settings reset

**Screenshots**

https://user-images.githubusercontent.com/68671029/124510301-1eb75c00-dd99-11eb-9c8a-76b4d3334f14.mp4

**Device information**
 - Device: Samsung Galaxy S6 Edge
 - OS: Android 7.0

**Additional context**
N/A

**Comment by vkmicro** [comment]
Reset button resets the tap'able checkmarks in the exercise 'sets'
see attached video.

https://user-images.githubusercontent.com/17531168/124640946-225fe700-de5c-11eb-978a-bebcfc345104.mp4

**Comment by Timbobeek** [comment]
Makes sense! Perhaps I would only have "reset" button visible once an exercise with sets is entered, and maybe locate the button right next to the sets blank squares? Because the button is still visible and tapable when there are no exercises present.
